Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -15734,6 +15764,11 @@ void WebPageProxy::didAdjustVisibilityWithSelectors(Vector<String>&& selectors)
     m_uiClient->didAdjustVisibilityWithSelectors(*this, WTFMove(selectors));
 }
 
+Ref<BrowsingContextGroup> WebPageProxy::protectedBrowsingContextGroup() const
```

Comment:
Why did this move?

Reply:
Ah I was just moving it in the same order that the functions are listed in the header (so after `didAdjustVisibilityWithSelectors`). Do we care about the order in the cpp file?

Reply:
> Do we care about the order in the cpp file?

No, so it didn't seem useful to increase patch size here.